internals/schemas/environment.conf.schema.json

```diff
@@ -193,6 +207,7 @@
         "STATIC_MAP_SERVER_URL",
         "TERMS_ENDPOINT",
         "USERS_ENDPOINT",
+        "areaTypeCodeForDistrict",
```

Comment:
This shouldn't be required. When `useAreasInsteadOfStadsdeel` is set to 'false', having to set a value here (that isn't used) doesn't make any sense.

Reply:
I see what you mean. However, it is required when `useAreasInsteadOfStadsdeel` is set to `true`, but it's not possible to specify it like that.

1. So, we either mark it as required and you indeed have to set it unnecessarily when `useAreasInsteadOfStadsdeel` is set to `false`.
2. Or, we don't mark it as required with the risk that it might not be defined even when `useAreasInsteadOfStadsdeel` is set to `true`. We would need to catch that case in the code and think about how to handle that situation.

I'm inclined to go for option 1 with the idea that we are moving towards `useAreasInsteadOfStadsdeel` being set to `true` always and us even removing the feature flag in the future.

Reply:
What about the solution proposed here: https://stackoverflow.com/questions/38717933/jsonschema-attribute-conditionally-required?

Reply:
Oh wow, I'll try that!